docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/upgradability-guidelines.md

```diff
@@ -13,6 +13,12 @@ related:
     link: docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-method-is-overridden-on-the-project-level.html
 ---
 
+Spryker has more than one thousand modules, and a bunch of them uses in each Spryker-based project. At the same time, the Sprykers team implements about 50 module updates each month. In this case, keeping your project up to date and receiving all necessary updates, coud take a lot of time for the project team.
+
+To improve the developer experience and decrease the time to keep your project up to date - we implemented the Upgrader and Integrator tools. These tools can automatically update vendors of the spryker-based projects and integrate changes to the project side.
```

Comment:
```To improve the developer experience and decrease the _amount of time, that is required_ to keep your project up to date...
```

Reply:
++

Replaced with the suggested structure